.github/workflows/go-postsubmit.yml

```diff
@@ -0,0 +1,40 @@
+name: GoPostSubmit
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
+  workflow_dispatch: {}
+
+env:
+  # Common versions
+  GO_VERSION: '1.16'
+  GO_REQUIRED_MIN_VERSION: ''
+  GOPATH: '/home/runner/work/governance-policy-propagator/governance-policy-propagator/go'
+defaults:
+  run:
+    working-directory: go/src/open-cluster-management.io/governance-policy-propagator
+
+jobs:
+  verify:
+    name: images
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 1
+          path: go/src/open-cluster-management.io/governance-policy-propagator
+      - name: install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: install imagebuilder
+        run: go install github.com/openshift/imagebuilder/cmd/imagebuilder@v1.2.1
```

Comment:
We'll have to compare what we're doing to other repos. This step doesn't look familiar to me.

Although, it does make me wonder: we're using the DockerHub golang image, so if we do this too many times we might hit the pull quota. We might need to move to a Red Hat builder image (which may be what this is).

Reply:
i copied this from the placement repo.